packages/server/README.md

```diff
@@ -35,15 +35,15 @@ Since this is slow, it's better to drive your development with tests.
 yarn build
 ```
 
-* `yarn lerna run test-unit --scope @packages/server --stream` executes unit tests in [`test/unit`](./test/unit)
-* `yarn lerna run test-integration --scope @packages/server --stream` executes integration tests in [`test/integration`](./test/integration)
-* `yarn lerna run test-performance --scope @packages/server --stream` executes performance tests in [`test/performance`](./test/performance)
-* `yarn lerna run test-e2e --scope @packages/server --stream` executes the large (slow) end to end tests in [`test/e2e`](./test/e2e)
+* `yarn test-unit` executes unit tests in [`test/unit`](./test/unit)
+* `yarn test-integration` executes integration tests in [`test/integration`](./test/integration)
+* `yarn test-performance` executes performance tests in [`test/performance`](./test/performance)
+* `yarn test-e2e` executes the large (slow) end to end tests in [`test/e2e`](./test/e2e)
 
-Each of these tasks can run in "watch" mode by appending `-watch` to the task:
+You can also use the `test-watch` command to rerun a test file whenever there is a change:
 
 ```bash
-yarn lerna run test-unit-watch --scope @packages/server --stream
+yarn test-watch /test/path/to/spec.js
 ```
 
```

Comment:
Can we add instructions for running individual tests and the globbing signature? I [added this in a not-yet merged PR](https://github.com/cypress-io/cypress/pull/6317/files#diff-2ad6c122ae558ed865b11e81e8917aadR51), so you could mostly take that and modify it for the new signature. It also has instructions for snapshot updating.